server/handlers/getSurvey.test.js

```diff
@@ -1,31 +1,20 @@
 /* eslint-disable no-undef */
-// const request = require('supertest');
-// const { initDb, closeDb } = require('../databaseConnection');
-// const app = require('../app');
+const request = require('supertest');
+const { initDb, closeDb } = require('../databaseConnection');
+const app = require('../app');
 
-// beforeEach(() => {
-//   return initDb();
-// });
-
-// afterEach(() => {
-//   return closeDb();
-// });
 
 describe('Testing GET /surveys/:id', () => {
-  it('Responds with JSON and status 200 when the survey has been found successfully.', (done) => {
-    done();
-    // const documentIdToFind = '507f1f77bcf86cd799439001';
-    // request(app)
-    //   .get(`/surveys/${documentIdToFind}`)
-    //   .set('Accept', 'application/json')
-    //   .expect(200)
-    //   .expect('Content-Type', /json/)
-    //   .end((err, res) => {
-    //     if (err) {
-    //       return done(err);
-    //     }
-    //     expect(res.body.auth).toEqual(true);
-    //     return done();
-    //   });
+  it('Responds with JSON and status 200 when the survey has been found successfully.', async () => {
+
+    await initDb();
+
+    const res = await request(app)
+      .get('/surveys/508f1f99bcf86cd799439014')
+      .set('Accept', 'application/json')
+      .expect('Content-Type', /json/)
+
+    expect(res.status).toEqual(200)
```

Comment:
Can we test the result being returned? If the status is 200, I expect something is sent back, could we make an assertion on this please?

Reply:
Had some issues getting that working so it's removed for now. Will do in the next iteration